Jenkinsfile

```diff
@@ -264,6 +264,13 @@ def runBuild(buildFlags, instrumentationTestTarget) {
         step([$class: 'LintPublisher'])
       }
     },
+    'JVM8 introExample check' : {
+      // Force build with JVM8, by disabling the cache, and check introExample.
+      sh """
+        cd examples/introExample
```

Comment:
If you should pick only one project to test, then maybe `moduleExample` would be the one to use as the _cross module model definition_ actually has different codepaths for JDK 8 and 11 in https://github.com/realm/realm-java/blob/master/realm/realm-annotations-processor/src/main/java/io/realm/processor/Utils.kt#L43 which is only triggered if the metadata is not immediately available in the model.

Reply:
Sounds like a good idea 👍